scalableminds/webknossos
# Isosurface frontend

### URL of deployed dev instance (used for testing):
- https://isosurfacefrontend.webknossos.xyz

### Steps to test:
- open a dataset with segmentation
- enable "render isosurfaces" in dataset settings
- move in the dataset and see the isosurfaces in the 3d viewport

### Issues:
- fixes #3440 

------
